NineChronicles.Headless/GraphTypes/States/Models/Table/StakeRegularRewardRowType.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace NineChronicles.Headless.GraphTypes.States.Models.Table
 {
-    public class StakeRegularRewardRowType : ObjectGraphType<StakeRegularRewardSheet.Row>
+    public class StakeRegularRewardRowType : ObjectGraphType<(int Level, long RequiredGold, StakeRegularRewardSheet.RewardInfo[] Rewards, StakeRegularFixedRewardSheet.RewardInfo[] BonusRewards)>
```

Comment:
I think it would be better to change the name & namespace for rewards rather than row.

Reply:
Yes, as you said, it can be confused because I think it can be understood as a `StakeRegularRewardSheet`'s row.

I'll rename it into `StakeRegularRewards` and `StakeRegularRewardsSheet` into `StakeRegularRewardsTable`. But `StakeRegularRewardsTable` seems not like good naming.

Do you have a naming suggestion about this? If you have, please share it 🙏🏻

Reply:
in the future, achieve reward also included in this type? I'd like to `StakeRewardsType`.

Reply:
Thanks for your suggestion! I applied the suggestion in [de92231](https://github.com/planetarium/NineChronicles.Headless/pull/1345/commits/de92231ad296beb59d663f882f97ecf110708ab7). Rename them and move to a new namespace called `Abstractions`, split from lib9c models, for high-level abstraction.